llvm/test/MC/RISCV/custom_reloc.s

```diff
@@ -33,3 +33,20 @@
   nop
   # CHECK-ASM: nop
   # CHECK-OBJ: addi zero, zero, 0x0
+
+  .reloc ., R_RISCV_VENDOR,    QUALCOMM
+  .reloc ., R_RISCV_QC_LO20_U, my_bar + 2
+  addi a1, a1, 0
+  # CHECK-ASM: [[L3:.L[^:]+]]:
+  # CHECK-ASM-NEXT: .reloc [[L3]], R_RISCV_VENDOR, QUALCOMM
+  # CHECK-ASM-NEXT: [[L4:.L[^:]+]]:
+  # CHECK-ASM-NEXT: .reloc [[L4]], R_RISCV_QC_LO20_U, my_bar+2
```

Comment:
The assembly streamer here just re-emits the string which it was passed from the directive parser in the call. I don't know if we want to do anything to normalise this into `R_RISCV_CUSTOM<n>` - it's a possibility.